include/ubiquity_motor/motor_parmeters.h

```diff
@@ -83,6 +102,18 @@ struct FirmwareParams {
             nh, "ubiquity_motor/pid_denominator", pid_denominator);
         pid_moving_buffer_size = getParamOrDefault(
             nh, "ubiquity_motor/window_size", pid_moving_buffer_size);
+        controller_board_version = getParamOrDefault(
+            nh, "ubiquity_motor/controller_board_version", controller_board_version);
+        estop_enable = getParamOrDefault(
+            nh, "ubiquity_motor/fw_estop_enable", estop_enable);
+        estop_pid_threshold = getParamOrDefault(
+            nh, "ubiquity_motor/fw_estop_pid_threshold", estop_pid_threshold);
+        max_speed_fwd = getParamOrDefault(
+            nh, "ubiquity_motor/fw_max_speed_fwd", max_speed_fwd);
+        max_speed_rev = getParamOrDefault(
+            nh, "ubiquity_motor/fw_max_speed_rev", max_speed_rev);
+        max_pwm = getParamOrDefault(
```

Comment:
I am not sure that we want to expose this value in a way that customers can fiddle with it. 
Is there a hard max in firmware to prevent someone from setting this at 100%?

Reply:
We do have a chance to wrong a past lack of speed limits and that may or may not be a good thing.
In the past the firmware sees the 32bit value and  upper 16 bits is one speed, lower 16 is other.  That of course is a possible desaster but was outside the scope of this fix.   This fix is to allow some lower value to be set.  If the user cranks this up to 0xfffe  for each one he is no worse off than the past.   Yes it is really not good but we have not addressed this issue and instead we place a limit in this way.  To revert to old max limits user can set these full on and the masking of 0xffff still is the old limit in that case.

Reply:
You have reminded me to follow up on something with the reverse limit so I'm going to re-inspect that and try some more tests prior to the next review which as you say is for me to simply push to this branch on remote and you will see update on the pull request.